### PR TITLE
test: removed toggleBar command and replaced with ts helper

### PR DESCRIPTION
**Pull request by NandanAnantharamu**
removed cy.toggleBar() from Commands.js file and replaced with ts helper method agHelper.CheckUncheck();

Link EE for runs - https://github.com/appsmithorg/appsmith-ee/pull/4311

/ok-to-test tags="@tag.All"
> [!TIP]
> 🟢 🟢 🟢 All cypress tests have passed! 🎉 🎉 🎉
> Workflow run: <https://github.com/appsmithorg/appsmith/actions/runs/9312018494>
> Commit: 04e2bdf96a95c1fb5706321200d03facbbb6a019
> Cypress dashboard url: <a href="https://internal.appsmith.com/app/cypress-dashboard/rundetails-65890b3c81d7400d08fa9ee5?branch=master&workflowId=9312018494&attempt=2" target="_blank">Click here!</a>

## Summary by CodeRabbit

- **Refactor**
  - Replaced `cy.togglebar` with `agHelper.CheckUncheck` across multiple test files for better consistency and maintainability in handling checkbox interactions.
  - Updated locators and selectors in various test files and configuration files to improve test reliability and readability.

- **Chores**
  - Updated spec names for limited tests in the Cypress test suite to align with new test structure and naming conventions.

### CODE REVIEW

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/ListV2/ListV2_PageNo_PageSize_spec.js [hunk 1]
```diff
@@ -142,27 +140,27 @@ describe(
...
       cy.get(commonlocators.bodyTextStyle)
         .first()
         .should("have.text", "PageSize 3");
 
       // toggle serversidepagination -> true
       cy.openPropertyPane("listwidgetv2");
-      cy.togglebar(".t--property-control-serversidepagination input");
+      _.agHelper.CheckUncheck(
+        ".t--property-control-serversidepagination input",
+      );
```
[the remark this answers is not in the document]
Comment (ApekshaBhosale) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/ListV2/ListV2_PageNo_PageSize_spec.js:156]: replace this .t--property-control-serversidepagination input with locator please 

serverSidePaginationCheckbox in commonLocators.json

**Review by ApekshaBhosale — COMMENTED** [review]
can we add this "t--property-control-allowclearingvalue input[type="checkbox"]" as locators as this has been used so many times in these specs?

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_ArrayField_spec.js [hunk 2]
```diff
@@ -375,7 +373,7 @@ describe(
       );
 
       // Enable Allow Country Code Change
-      cy.togglebar(
+      agHelper.CheckUncheck(
         ".t--property-control-allowcurrencychange input[type='checkbox']",
```
Comment (ApekshaBhosale) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_ArrayField_spec.js:377]: use a locator for this please? allowCurrencyChange from commonLocators file

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_FieldProperties_1_spec.js [hunk 3]
```diff
@@ -125,7 +125,7 @@ describe(
     it("6. disables field when disabled switched on and when autofill is disabled we should see the autofill attribute in the input field", () => {
       EditorNavigation.SelectEntityByName("JSONForm1", EntityType.Widget);
       cy.openFieldConfiguration("name");
-      cy.togglebar(`.t--property-control-disabled input`);
+      agHelper.CheckUncheck(".t--property-control-disabled input");
```
Comment (ApekshaBhosale) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_FieldProperties_1_spec.js:128]: this locators has been used multiple times. let's create a variable and use it across please

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_FieldProperties_2_spec.js [hunk 4]
```diff
@@ -195,7 +195,7 @@ describe(
       cy.togglebarDisable(`.t--property-control-visible input`);
       cy.get(`${fieldPrefix}-radio`).should("not.exist");
       cy.wait(500);
-      cy.togglebar(`.t--property-control-visible input`);
+      agHelper.CheckUncheck(".t--property-control-visible input");
```
Comment (ApekshaBhosale) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_FieldProperties_2_spec.js:198]: let's add variable for this as well

**Review by ApekshaBhosale — CHANGES_REQUESTED** [review]
let's move as many locators to locators file instead of using them directly @NandanAnantharamu

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_FieldProperties_2_spec.js (hunk 4 above)
Comment (NandanAnantharamu) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_FieldProperties_2_spec.js]: done

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/Multiselect/MultiSelect2_spec.js [hunk 5]
```diff
@@ -105,7 +105,7 @@ describe(
 
     it("4. Does not clear the search field when widget is closed and serverSideFiltering is on", () => {
       // Turn on server side filtering for the widget
-      cy.togglebar(
+      _.agHelper.CheckUncheck(
         '.t--property-control-serversidefiltering input[type="checkbox"]',
```
[the remark this answers is not in the document]
Comment (NandanAnantharamu) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/Multiselect/MultiSelect2_spec.js:107]: done

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_FieldProperties_1_spec.js (hunk 3 above)
Comment (NandanAnantharamu) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_FieldProperties_1_spec.js]: done

File: app/client/cypress/limited-tests.txt [hunk 6]
```diff
@@ -1,5 +1,13 @@
 # To run only limited tests - give the spec names in below format:
-cypress/e2e/Regression/ClientSide/Templates/Fork_Template_spec.js
+cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_ArrayField_spec.js
+cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_FieldProperties_1_spec.js
+cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_FieldProperties_2_spec.js
+cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_HiddenFields_1_spec.js
+cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_HiddenFields_2_spec.js
```
Comment (ApekshaBhosale) [inline app/client/cypress/limited-tests.txt:6]: revert this file

**Review by ApekshaBhosale — CHANGES_REQUESTED** [review]

File: app/client/cypress/limited-tests.txt (hunk 6 above)
Comment (NandanAnantharamu) [reply app/client/cypress/limited-tests.txt]: done

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_ArrayField_spec.js [hunk 7]
```diff
@@ -375,7 +370,7 @@ describe(
...
```
Comment (ApekshaBhosale) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_ArrayField_spec.js:374]: this is missed. we already have locator for this. please add it

**Review by ApekshaBhosale — CHANGES_REQUESTED** [review]

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_ArrayField_spec.js (hunk 7 above)
Comment (NandanAnantharamu) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/JSONForm/JSONForm_ArrayField_spec.js]: done

**Review by ApekshaBhosale — APPROVED** [review]

**Pull request merged**